packages/rancher-monitoring/templates/crd-template/templates/jobs.yaml

```diff
@@ -49,8 +49,12 @@ spec:
           - -c
           - >
             echo "Applying CRDs...";
-            kubectl apply -f /etc/config/crd-manifest.yaml;
-            
+            mkdir -p /etc/crd-manifest;
+            base64 -d /etc/config/crd-manifest.tgz > /etc/crd-manifest/crd-manifest.tgz;
+            tar -xzvf /etc/crd-manifest/crd-manifest.tgz -C /etc/crd-manifest;
+            rm /etc/crd-manifest/crd-manifest.tgz;
+            kubectl apply -Rf /etc/crd-manifest;
```

Comment:
can we just replace this with:

```bash
base64 -d /etc/config/crd-manifest.tgz | tar -xvzfO | kubectl apply -
```

Reply:
what does `-Rf` do in the kubectl commmand?

Reply:
The section is updated to reduce unnecessary steps, but we can achieve it in one pipeline because there are directory struct information in the output of `tar` command that makes  `kubectl` fail.